README.md

```diff
@@ -9,7 +9,7 @@ The Snowflake .NET connector supports the the following .NET framework and libra
 
 - .NET Framework 4.7.2
 - .NET Framework 4.7.3
-- .NET Core 2.0
+- .NET Core 6.0
```

Comment:
This file also contains some examples that appear to refer to .NET Core 2.0:

```
dotnet test -f netcoreapp2.0
```

```
OpenCover.4.6.519\tools\OpenCover.Console.exe ... output:"netcoreapp2.0_coverage.xml" ...
```

Do these references need to be updated as well?

Reply:
Fixed. Thanks.